pageserver/src/tenant/remote_timeline_client/index.rs

```diff
@@ -55,7 +48,7 @@ impl LayerFileMetadata {
         let mut changed = false;
```

Comment:
Is this function unused now?

Reply:
Thank you, yes, removed along with the comment spotted above.